azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/database/DatabaseService.java

```diff
@@ -97,6 +103,31 @@ public void restoreNamespaceRoleAccess(String namespaceRole) {
     databaseDao.restoreLoginPrivileges(namespaceRole);
   }
 
+  private SecretKey decodeBase64EncryptionKey(String encryptionKeyBase64) {
```

Comment:
What is the difference between this and the method just below?

Reply:
This one was copy/pasted and unused. Now removed.